index.html

```diff
@@ -20,24 +20,25 @@
         </li>
       </ul>
     </div>
-    <br>
+    <br />
     <section class="header-container">
       <div class="sub-container">
         <h1 class="header-highlight-1">Hey There.I’m OMSOO</h1>
         <h3 class="header-highlight-2">I’am a Software Developer</h3>
         <p class="header-highlight-3">
-          I can help you build a product , feature or website Look through some of
-          my work and experience! If you like what you see and have a project you
-          need coded, don’t hestiate to contact me.
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
         </p>
-        <span class="social-icons"> 
-          <span class="fab fa-linkedin-in"></i> 
+        <span class="social-icons">
+          <span class="fab fa-linkedin-in">
             <i class="fab fa-twitter"></i>
-            <i class="fab fa-snapchat"></i>
+            <i class="fab fa-angellist"></i>
             <i class="fab fa-pinterest-p"></i>
-            <i class="fab fa-youtube"></i>
+            <i class="fab fa-github"></i>
           </span>
```

Comment:
- [ ] Kindly this is a list so it'll be more semantic to use `ul` tag to group them instead of `span` and `li` per icon, and each icon has a link (`a` tag).
i.e. :

```
<ul>
     <li><i class=""></i></li>
      ......
</ul>
```